src/Daemon/BaseDaemon.cpp

```diff
@@ -104,7 +104,8 @@ static const size_t signal_pipe_buf_size =
     + sizeof(ucontext_t*)
     + sizeof(StackTrace)
     + sizeof(UInt32)
-    + sizeof(void*);
+    + sizeof(void*)
+    + sizeof(std::vector<StackTrace::FramePointers>);
```

Comment:
This is actually wrong, because we write the size of vector and then each member as binary.

@yakov-olkhovskiy any ideas what to do?

Reply:
yes I think you are right
trying to debug a segfault in signal handler so I pushed this instantly when I noticed this
we don't have to do anything then I guess

Reply:
Do we want to maybe at least add space for size of vector so it avoids extra flush if we don't write frame pointers?

Reply:
I think we still can increase size of this buffer by `UInt` size (for empty vector) so when feature is not enabled buffer will be efficient

Reply:
yes, that's what I meant by my comment. Okay will do that and change the description.